components/landing/TheQiskitCapabilitiesSection/index.vue

```diff
@@ -1,16 +1,235 @@
+      link: {
+        url: 'https://qiskit.org/documentation/apidoc/circuit_library.html',
+        label: 'Learn more',
+        segment: { cta: 'feature-circuit', location: 'homepage-capabilities-section' }
+      }
+    },
+    {
+      title: 'Transpiler',
+      description: 'The transpiler translates Qiskit code into an optimized circuit using a backend’s native gate set, allowing users to program for any quantum processor or processor architecture with minimal inputs.',
+      thumbnailResource: '/images/landing-page/transpiler.png',
+      link: {
+        url: 'https://qiskit.org/documentation/apidoc/transpiler.html',
+        label: 'Learn more',
+        segment: { cta: 'transpiler', location: 'homepage-capabilities-section' }
+      }
+    },
+    {
+      title: 'Run on real hardware',
+      description: 'Users can run and schedule jobs on real quantum processors, and employ Qiskit Runtime to orchestrate quantum programs on cloud-based CPUs, QPUs, and GPUs.',
+      thumbnailResource: '/images/library/chip-01.png',
+      link: {
+        url: 'https://qiskit.org/documentation/apidoc/ibmq_provider.html',
+        label: 'Learn more',
+        segment: { cta: 'run-on-hardware', location: 'homepage-capabilities-section' }
+      }
+    },
+    {
+      title: 'Try it yourself',
+      description: 'Ready to explore Qiskit’s capabilities for yourself? Copy the code to the right of this paragraph and try running it in your local Python environment. You can also click the link to the IBM Quantum Lab and test it there.',
+      thumbnailResource: '/images/landing-page/med_02_1.png'
+    }
+  ]
+
+  quantumLabCTA = {
+    url: 'https://quantum-computing.ibm.com/lab/files/qiskit-textbook/getting-started/example.ipynb',
+    label: 'Open in Quantum Lab',
+    segment: { cta: 'quantum-lab', location: 'homepage-capabilities-section' }
+  }
+
+  copyCodeCTA = {
+    url: '',
+    label: 'Copy code',
+    segment: { cta: 'copy-code', location: 'homepage-capabilities-section' }
+  }
+
+  isActiveSection (item: QiskitCapability, index: number): boolean {
+    if (this.capabilities[this.capabilities.length - 1].title === this.activeSection) {
+      return true
+    }
+
+    return item.title === this.activeSection || (this.activeSection === '' && index === 0)
+  }
+
+  copyToClipboard (): void {
+    const codeSnippet = document.querySelector('.code-cell')?.textContent!
+    navigator.clipboard.writeText(codeSnippet)
+  }
 }
 </script>
+
+<style lang="scss" scoped>
+@import '~carbon-components/scss/globals/scss/typography';
+
+$cta-max-width: 14rem;
+
+.qiskit-capabilities-section {
+  padding-bottom: $spacing-09;
+
+  &__container {
+    display: flex;
+    padding-top: $spacing-05;
+    padding-bottom: $spacing-09;
+
+    @include mq($until: medium) {
+      padding-bottom: $spacing-05;
+    }
+  }
+
+  &__capabilities {
+    display: flex;
+    align-items: flex-start;
+    width: 100%;
```

Comment:
I believe this has no noticeable effect, but let me know if I'm missing something.

```suggestion
```